### PR TITLE
add header in block with image

### PR DESCRIPTION
**Pull request by denmanma**

### CODE REVIEW

File: resume.html [hunk 1]
```diff
@@ -5,6 +5,9 @@
 		Resume: MICHAEL DENMAN
 	</title>
 	<style>
+		* {
+			box-sizing: border-box;
+		}
```
Comment (jonathan-irvin) [inline resume.html]: This is too broad, be more specific?

File: resume.html [hunk 2]
```diff
@@ -37,22 +40,29 @@
...
+				<a id="linkedin-link" href='https://www.linkedin.com/in/michael-denman/'> Check out my LinkedIn</a><br>
+			</address>
+			United States Citizen, Languages: Spanish & English<br>
+			<a id="sec+-link"
+				href="https://www.youracclaim.com/badges/1c4d9cb9-b9da-4602-b60a-48be5a33ad94">Security+</a> &
+			<a id="casp+-link" href="https://www.youracclaim.com/badges/b698b10a-25f6-4f20-9d5a-5455360fffba">CASP+</a>
+			Certified, Job Availability Date: 23 Sept 2019<br>
+		</div>
+
+	</div>
```
Comment (denmanma) [inline resume.html]: Trying to get all this content centered and in it's own box

File: resume.html [hunk 3]
```diff
@@ -5,6 +5,9 @@
 		Resume: MICHAEL DENMAN
 	</title>
 	<style>
+		* {
+			box-sizing: border-box;
+		}
 		body {
 			background-color: #333333;
 			color: #cccccc;
```
Comment (jonathan-irvin) [inline resume.html]: Let's change these up to only style elements on your page.  Reserve body-wide styles for last.

File: resume.html [hunk 4]
```diff
@@ -37,22 +40,29 @@
...
-	United States Citizen, Languages: Spanish & English<br>
-	<a id="sec+-link" href="https://www.youracclaim.com/badges/1c4d9cb9-b9da-4602-b60a-48be5a33ad94">Security+</a> &
-	<a id="casp+-link" href="https://www.youracclaim.com/badges/b698b10a-25f6-4f20-9d5a-5455360fffba">CASP+</a>
-	Certified, Job Availability Date: 23 Sept 2019<br>
-
+	<div style="background-color: #000000; max-width: 600px;">
+		<div style="align-content:flex-start;">
+			<img id="self-photo" src="assets\images\self-photo.png" alt="Self-photo" style="vertical-align: top; float:left;">
+		</div>
+		<div style="float:inline-end;">
```
Comment (jonathan-irvin) [inline resume.html]: Move all styles to css

File: resume.html [hunk 5]
```diff
@@ -37,22 +40,29 @@
...
-		--
-		<a id="linkedin-link" href='https://www.linkedin.com/in/michael-denman/'> Check out my LinkedIn</a><br>
-	</address>
-	United States Citizen, Languages: Spanish & English<br>
-	<a id="sec+-link" href="https://www.youracclaim.com/badges/1c4d9cb9-b9da-4602-b60a-48be5a33ad94">Security+</a> &
-	<a id="casp+-link" href="https://www.youracclaim.com/badges/b698b10a-25f6-4f20-9d5a-5455360fffba">CASP+</a>
-	Certified, Job Availability Date: 23 Sept 2019<br>
-
+	<div style="background-color: #000000; max-width: 600px;">
+		<div style="align-content:flex-start;">
```
Comment (jonathan-irvin) [inline resume.html]: Move all styles to css

File: resume.html [hunk 6]
```diff
@@ -37,22 +40,29 @@
...
+			<img id="self-photo" src="assets\images\self-photo.png" alt="Self-photo" style="vertical-align: top; float:left;">
+		</div>
+		<div style="float:inline-end;">
+			<h2>MICHAEL ALEXANDER DENMAN</h2>
+			<address>
```
Comment (jonathan-irvin) [inline resume.html:24]: What's this for?

File: resume.html [hunk 7]
```diff
@@ -37,22 +40,29 @@
...
+				<a id="phone-number" href="tel:+13252772340">Call Me</a>
```
Comment (jonathan-irvin) [inline resume.html:25]: Good use of tel:

File: resume.html (hunk 2 above)
Comment (jonathan-irvin) [reply resume.html]: @denmanma It's time to play a game! https://flexboxfroggy.com/

**Pull request merged**